### PR TITLE
NXP-24518: add S3 Direct Upload marketplace

### PR DESCRIPTION
**Pull request by Gabez0r**
Depends on:
- nuxeo/addons#46
- nuxeo/nuxeo-core-binarymanager-cloud#27

### CODE REVIEW

**Comment by afjusto** [comment]
👍

**Comment by kevinleturc** [comment]
👍

**Comment by driverpt** [comment]
👍

File: marketplace/pom.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.marketplace.amazon-s3-direct-upload</groupId>
+    <artifactId>amazon-s3-direct-upload-parent</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (Gabez0r) [inline marketplace/pom.xml]: Should we use 1.0.0 for LTS2017 and use 1.1.0 for master?

**Pull request merged**